en/config/actuators.md

```diff
@@ -45,8 +46,7 @@ This includes the number and position of [motors](#motor-geometry), and the numb
 For VTOL tiltrotor vehicles, it will also include the number and properties of [tilt servos](#motor-tilt-servo-geometry)
 
 :::note
-The UI is customised for the selected airframe.
-Only the motors and/or control surface options that are configurable for the airframe type are displayed.
+The UI shows the configured motor positions, customized to your vehicle (Currently only for Multicopter)
```

Comment:
This change is wrong. This is about the whole geometry section not the multicopter image. 

Or by example, if you have a vehicle with motors that you can't customise the geometry on, they won't be displayed."

I will revert.

Reply:
Actually, I'm going to merge the two points :-)